framework/auth/views.py

```diff
@@ -655,14 +655,6 @@ def external_login_confirm_email_get(auth, uid, token):
     service_url = request.url
 
     if external_status == 'CREATE':
-        mails.send_mail(
```

Comment:
There is another place that sends out emails. It's on L746 in `confirm_email_get`. I believe that's what's sending out the emails on staging3 right now.

Reply:
@brianjgeiger removed that bit.